tgui/packages/tgui/interfaces/_FaxMachine.js

```diff
@@ -0,0 +1,286 @@
+          <Stack vertical height={15}>
+            <Stack.Item height={2}>
+              <Tabs>
+                <Tabs.Tab
+                  width="50%"
+                  icon="copy"
+                  selected={tab === 1}
+                  onClick={() => setTab(1)}>
+                  <b>Send A Fax</b>
+                </Tabs.Tab>
+                <Tabs.Tab
+                  width="50%"
+                  icon="broadcast-tower"
+                  selected={tab === 2}
+                  onClick={() => {
+                    setTab(2);
+                    act('read_last_received');
+                  }}>
+                  <Stack grow width="100%">
+                    <Stack.Item grow textAlign="left">
+                      <b>received Faxes </b>
+                    </Stack.Item>
+                    {received_paper && !!unread_message && (
+                      <Stack.Item grow textAlign="right">
+                        <i>{"New message!"}</i>
+                      </Stack.Item>)}
+                  </Stack>
+                </Tabs.Tab>
+              </Tabs>
+            </Stack.Item>
+            <Stack.Item mt={1} height={13}>
+              {tab === 1 && (
+                stored_paper ? (
+                  <Box mt={1}>
+                    <span style={{
+                      color: (emagged ? "lightgreen" : "lightblue"),
+                      fontWeight: "bold" }}>
+                      Message to Send:
+                    </span>
+                    <BlockQuote mt={1}>
+                      {stored_paper.contents}
+                    </BlockQuote>
+                  </Box>
+                ) : (
+                  <Box>
+                    <i>
+                      Insert a paper into the machine
+                      to fax its contents somewhere.
+                    </i>
+                  </Box>
+                ))}
+              {tab === 2 && (
+                received_paper ? (
+                  <Box mt={1}>
+                    <span style={{ color: "gold", fontWeight: "bold" }}>
+                      Message from {received_paper.source}:
+                    </span>
+                    <BlockQuote mt={1}>
+                      {received_paper.contents}
+                    </BlockQuote>
+                  </Box>
+                ) : (
+                  <Box>
+                    <i>
+                      No papers have been received.
+                    </i>
+                  </Box>
+                ))}
+            </Stack.Item>
+            <Stack.Item>
+              {tab === 1 && stored_paper && (
+                <Stack>
+                  <Stack.Item>
+                    <Button
+                      height="100%"
+                      icon="fax"
+                      color={emagged
+                        ? "bad" : "good"}
+                      content="Send to: "
+                      disabled={tab !== 1
+                        || !stored_paper
```

Comment:
I can't imagine these two conditions in `disabled` would ever happen considering the button is only actually visible if they pass above on L133, but may as well keep them just in case they were added deliberately.